src/Commands/ChurnCommand.php

```diff
@@ -162,10 +172,54 @@ protected function displayResults(OutputInterface $output, ResultCollection $res
 
         $table = new Table($output);
         $table->setHeaders(['File', 'Times Changed', 'Complexity', 'Score']);
-        foreach ($results->orderByScoreDesc()->take($this->config->getFilesToShow()) as $result) {
-            $table->addRow($result->toArray());
-        }
+
+        $table->addRows($this->handleResults($results));
+
         $table->render();
-        echo "  " . $this->filesCount . " files analysed in {$totalTime} seconds using " . $this->config->getParallelJobs() .  " parallel jobs.\n\n";
+        echo "  "
+            . $this->filesCount
+            . " files analysed in {$totalTime} seconds using "
+            . $this->config->getParallelJobs()
+            . " parallel jobs.\n\n";
+    }
+
+    /**
+     * Handle the results.
+     * @param  ResultCollection $results
+     * @return array
+     */
+    public function handleResults(ResultCollection $results): array
```

Comment:
Do you think this method could be extracted to be within the `ResultCollection` itself rather than here inside the `ChurnCommand` class? I think that would be better if it can be done.

Reply:
That would also remove the `minScoreExists` and `filterByMinScore` from this class.

Reply:
getting on it.